.agents/commands/review/references/verdict-format.md

```
@@ -0,0 +1,29 @@
+# Verdict Format
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)